src/main/java/org/openelisglobal/common/rest/bean/ViewNonConformEventsResponse.java

```diff
@@ -0,0 +1,126 @@
+package org.openelisglobal.common.rest.bean;
+
+import java.util.List;
+
+import org.openelisglobal.common.util.IdValuePair;
+import org.openelisglobal.qaevent.valueholder.NcEvent;
+import org.openelisglobal.qaevent.valueholder.NceCategory;
+import org.openelisglobal.qaevent.valueholder.NceType;
+import org.openelisglobal.sampleitem.valueholder.SampleItem;
+import org.openelisglobal.systemuser.valueholder.SystemUser;
+
+
+public class ViewNonConformEventsResponse {
+
+      private List<NcEvent> results;
```

Comment:
Good work here, Something small, Align line 15 with other lines. it is not well formated.